music21/stream/makeNotation.py

```diff
@@ -1708,6 +1709,68 @@ def makeAccidentalsInMeasureStream(
     if isinstance(s, Stream):
         s.streamStatus.accidentals = True
 
+def ornamentalPitches(s: StreamType) -> list[pitch.Pitch]:
+    '''
+    Returns all ornamental :class:`~music21.pitch.Pitch` objects found in any
+    ornaments in notes/chords in the stream (and substreams) as a Python list.
+
+    Very much like the pitches property, except that instead of returning all
+    the pitches found in notes and chords, it returns the ornamental pitches
+    found in the ornaments on the notes and chords.
+
+    If you want a list of _all_ the pitches in a stream, including the ornamental
+    pitches, you can call s.pitches and makeNotation.ornamentalPitches(s),
+    and then combine the two resulting lists into one big list.
+    '''
+    from music21.stream import Stream
+    post = []
+    for e in s.elements:
+        if isinstance(e, Stream):
```

Comment:
this can be done without the import with `if 'Stream' in e.classes` which is why .classes is still there.  NBD!